testsuite/src/it/java/io/quarkus/ts/startstop/utils/Commands.java

```diff
@@ -158,6 +160,8 @@ public static String getQuarkusVersion() {
     public static String getQuarkusGroupId() {
         if (getQuarkusVersion().contains(REDHAT_VERSION_TAG)) {
             return QUARKUS_PLATFORM_GROUP_ID;
+        } else if (getQuarkusVersion().contains(QUARKUS_MAIN_VERSION)) {
```

Comment:
Covers also branches case - e.g. `3.2.999-SNAPSHOT` https://github.com/quarkusio/quarkus/blob/3.2/pom.xml#L10C18-L10C34

Reply:
yeah, I know, that was my first thought :-)